src/example/java/com/box/sdk/example/SearchExamplesAsAppUser.java

```diff
@@ -0,0 +1,346 @@
+            fullSizeOfResult = searchResults.fullSize();
+
+            print("offset: " + offset + " of fullSizeOfResult: " + fullSizeOfResult);
+            printSearchResults(searchResults);
+
+            offset += limit;
+        }
+
+
+        print("******Owner Id's Filter Search******");
+        List<String> ownerUserIds = new ArrayList<String>();
+        ownerUserIds.add("268117237");
+
+
+        bsp.clearParameters();
+        bsp.setQuery("Assemble");
+        bsp.setOwnerUserIds(ownerUserIds);
+
+        searchResults = bs.search(0, 1000, bsp);
+        printSearchResults(searchResults);
+
+        print("******Metadata Search Filter Tests******");
+        testMetaDataFilter(bsp, bs);
+
+
+        print("*****Description Search******");
+        List<String> contentTypes = new ArrayList<String>();
+        contentTypes.add("description");
+
+        bsp.clearParameters();
+        bsp.setContentTypes(contentTypes);
+        bsp.setQuery("test");
+
+        searchResults = bs.search(0, 1000, bsp);
+        printSearchResults(searchResults);
+
+
+
+        print("*****Created Date Range Search******");
+        bsp.clearParameters();
+        try {
+            SimpleDateFormat sdf = new SimpleDateFormat("M-dd-yyyy hh:mm:ss");
+            String createdAtRangeFromDateString = "01-31-2016 00:00:00";
+            String createdAtRangeToDateString = "04-01-2016 00:00:00";
+
+            Date createdAtRangeFromDate = sdf.parse(createdAtRangeFromDateString);
+            Date createdAtRangeToDate = sdf.parse(createdAtRangeToDateString);
+
+
+            bsp.setCreatedAtRangeFromDate(createdAtRangeFromDate);
+            bsp.setCreatedAtRangeToDate(createdAtRangeToDate);
+            bsp.setQuery("File");
+
+            searchResults = bs.search(0, 1000, bsp);
+            printSearchResults(searchResults);
+
+        } catch (ParseException e) {
+            // TODO Auto-generated catch block
+            e.printStackTrace();
+        }
+
+        print("*****Updated Date Range Search******");
+        bsp.clearParameters();
+        try {
+            SimpleDateFormat sdf = new SimpleDateFormat("M-dd-yyyy hh:mm:ss");
+            String updatedAtRangeFromDateString = "01-31-2016 00:00:00";
+            String updatedAtRangeToDateString = "05-01-2016 00:00:00";
+
+            Date updatedAtRangeFromDate = sdf.parse(updatedAtRangeFromDateString);
+            Date updatedAtRangeToDate = sdf.parse(updatedAtRangeToDateString);
+
+
+            bsp.setUpdatedAtRangeFromDate(updatedAtRangeFromDate);
+            bsp.setUpdatedAtRangeToDate(updatedAtRangeToDate);
+            bsp.setQuery("File");
+
+            searchResults = bs.search(0, 1000, bsp);
+            printSearchResults(searchResults);
+
+        } catch (ParseException e) {
+            // TODO Auto-generated catch block
+            e.printStackTrace();
+        }
+
+    }
+
+    public static void testMetaDataFilter(BoxSearchParameters bsp, BoxSearch bs) {
```

Comment:
I'd suggest breaking up each section of this method into separate methods. The "test" prefix is also kind of confusing because that's the naming convention for unit tests.